packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.stories.tsx

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ResetPasswordWithRecoveryKeyVerified from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { useNavigate } from '@reach/router';
+
+export default {
+  title: 'pages/ResetPasswordWithRecoveryKeyVerified',
+  component: ResetPasswordWithRecoveryKeyVerified,
+} as Meta;
+
+export const Default = () => (
```

Comment:
If you add any other states in the future (applicable to other files too), we should create something like `storyWithContext` or `storyWithProps` done in other storybook files so we don't have to keep setting the wrappers. This seems fine in this file though.